src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/LeggTilEllerFjernBrevmottakere/useLeggTilFjernBrevmottaker.ts

```diff
@@ -45,14 +50,46 @@ export interface IRestBrevmottaker {
 
 const useLeggTilFjernBrevmottaker = () => {
     const { settToast } = useApp();
-    const { settÅpenBehandling } = useBehandling();
+    const { åpenBehandling: åpenBehandlingRessurs, settÅpenBehandling } = useBehandling();
     const { behandlingId } = useSakOgBehandlingParams();
     const { request } = useHttp();
+    const åpenBehandling = hentDataFraRessurs(åpenBehandlingRessurs);
 
     const mottaker = useFelt<Mottaker | ''>({
         verdi: '',
-        valideringsfunksjon: felt =>
-            felt.verdi !== '' ? ok(felt) : feil(felt, 'Feltet er påkrevd'),
+        avhengigheter: { åpenBehandling },
+        valideringsfunksjon: (felt, avhengigheter) => {
+            const eksisterendeMottakere: IRestBrevmottaker[] =
```

Comment:
`avhengigheter` har ikke eksplisitt typing av sine attributter, så caster her for å ha riktig typing i resten av funksjonen